src/game.js

```diff
@@ -0,0 +1,117 @@
+import Tile from './tiles';
+
+class Game {
+  constructor(size) {
+    const tilesWin = [];
+    this.tilesWin = tilesWin;
+    this.length = size * size;
+   
+    this.bindMethods();
+  
+  }
+  getWinnerTiles() {
+    for (let i = 0; i < this.length - 1; i++) {
```

Comment:
Also here we can start iteration from the for (let i = 1; i < this.length; i++) ` probably it will look more clear. 
But as you have done also fine. so you can leave as it is

Reply:
I agree with you especial with method push();
Done